### DISCUSSION TITLE
Fixed width of SendLater colum with userChrome.css?

### DISCUSSION THREAD

**Discussion by ghost** [Extended-Thunder/send-later#394] [Q&A]
I use custom CSS code in the userChrome.css config file to set a fixed width for several columns, for example this one to set the width of the "Date" column to 10em:
`#dateCol {
	max-width: 10em !important;
	min-width: 10em !important; }`

My question: Is it possible to set a fixed width for the "SendLater" column, too? I have tried the following code, but it did not work for me:
`#sendlater3-colXSendLaterAt {
	max-width: 10em !important;
	min-width: 10em !important; }`

**Comment by jph-sendlater** [comment]
Good question. In the latest release (9.x), I believe the column's id is different than that, but I'm on mobile right now and would have to check that when I'm back at my desktop. You should poke around to make sure you have the right ID.
